frontend/src/components/RFI/NMF.vue

```diff
@@ -151,19 +125,22 @@
 
 <script>
 import { mapActions, mapState } from 'pinia';
-import { useAppStore } from '../../store/app.js';
-import { useApplicationStore } from '../../store/application.js';
-import { useNmfAppStore } from '../../store/nmfApp.js';
-import { useNavBarStore } from '../../store/navBar.js';
 
-import alertMixin from '../../mixins/alertMixin.js';
-import NavButton from '../../components/util/NavButton.vue';
-import FacilityHeader from '../guiComponents/FacilityHeader.vue';
+import NavButton from '@/components/util/NavButton.vue';
+import FacilityHeader from '@/components/guiComponents/FacilityHeader.vue';
+
+import { useAppStore } from '@/store/app.js';
+import { useApplicationStore } from '@/store/application.js';
+import { useNmfAppStore } from '@/store/nmfApp.js';
+import { useNavBarStore } from '@/store/navBar.js';
+
+import alertMixin from '@/mixins/alertMixin.js';
+import rules from '@/utils/rules.js';
 
 let model = { x: [] };
```

Comment:
Do we need this here? Could we remove this and just declare model in data()?

Reply:
it's updated.